python/wpt/exporter/__init__.py

```diff
@@ -184,6 +184,7 @@ def run(self, payload: dict, step_callback=None) -> bool:
         try:
             servo_pr = self.servo.get_pull_request(pull_data["number"])
             downstream_wpt_branch = self.downstream_wpt.get_branch(
+                # pyrefly: ignore  # bad-argument-type
```

Comment:
Could we actually convert the `PullRequest.pr` to a `str` here? This seems like a valid issue.